prow/jobs/kyma/tests/event-bus/test-event-bus.yaml

```diff
@@ -0,0 +1,45 @@
+job_template: &job_template
+  skip_report: true
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/kyma/tests/event-bus"
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository: "true"
+
+presubmits: # runs on PRs
+  kyma-project/kyma:
+    - name: kyma-tests-event-bus
+      run_if_changed: "^tests/event-bus/"
```

Comment:
after update Prow, now we are able to specify run_if_changed on postsubmit job also.

Reply:
@aszecowka Thanks for your review. Please check now, I added the option to the `postsubmit` job. However, I'd be glad if you tell me how exactly would be the behavior, it's expected to me when it comes to the `presubmit`, not the `postsubmit`. Thanks!